packages/components/src/components/form/SegmentedButton/SegmentedButton.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button as ChakraButton, ResponsiveValue, useMultiStyleConfig } from '@chakra-ui/react';
+import { Flex, IconProps } from '@components';
+import { useResponsiveValue } from '@hooks';
+import React, { FC, ReactElement, cloneElement, isValidElement, useState } from 'react';
+import {
+  ButtonProps,
+  OptionValue,
+  SegmentedButtonProps,
+  SegmentedButtonSize,
+  SegmentedButtonStaticMembers,
+  SegmentedButtonStyles,
+  SegmentedIconButtonProps,
+} from './types';
+
+export const SegmentedButton: FC<SegmentedButtonProps> & SegmentedButtonStaticMembers = ({
```

Comment:
is it possible to export this as a forwardReference? Not a _hard requirement_ but definitely a nice to have

Reply:
Done it